lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -122,6 +122,21 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       element.appendChild(this.renderCategoryHeader(category, groups));
     }
 
+    // Budgets
+    const budgetAudit = category.auditRefs.find((audit) => audit.id === 'resource-budget');
+    if (budgetAudit && budgetAudit.result && budgetAudit.result.details) {
```

Comment:
```suggestion
    if (budgetAudit && budgetAudit.result.details) {
```
(`budgetAudit.result` is always defined if `budgetAudit` is)